tests/samplers_tests/test_gp.py

```diff
@@ -9,6 +9,7 @@
 
 import optuna
 import optuna._gp.acqf as acqf_module
+import optuna._gp.gp as optuna_gp
```

Comment:
```suggestion
import optuna._gp.gp as gp
```
How about naming like this, following the import in `sampler.py`.

https://github.com/optuna/optuna/blob/255fc3de0d64e7534bed24df2bfd0d4450bfb9a7/optuna/samplers/_gp/sampler.py#L29

Reply:
Let me keep the module name as is to avoid confusion :bow: